src/plugins/intel_gpu/src/runtime/ocl/ocl_memory.cpp

```diff
@@ -3,13 +3,15 @@
 //
 
 #include "intel_gpu/runtime/debug_configuration.hpp"
+#include "intel_gpu/runtime/device.hpp"
```

Comment:
[random spot]
Shouldn't we apply the same extended limit for [OpenCL buffers](https://github.com/intel/compute-runtime/blob/master/programmers-guide/ALLOCATIONS_GREATER_THAN_4GB.md#opencl)?
```
cl_mem_flags flags = 0;
flags |= CL_MEM_ALLOW_UNRESTRICTED_SIZE_INTEL;

cl_mem buffer = clCreateBuffer(context, flags, size, host_ptr, errcode_ret);
```